### DISCUSSION TITLE
I'm a noob and need help

### DISCUSSION THREAD

**Discussion by iEdier** [UltimaHoarder/UltimaScraper#1957] [Q&A]
Hey guys, hope you're all doing well.

I just found out about this code and it's the first time I'm using Github and Python so the instructions are confusing to me, hopefully somebody can guide me on what am I doing wrong :(
I was able to get the "Poetry" installation on my command prompt but I get roadblocked on the next step, when I run the "python start_ofd.py" I was getting an error which I initially fixed, however I get stuck in: 

"from yarl import URL
ModuleNotFoundError: No module named 'yarl'"

I'm not sure on what this means or what can I do to run the program, I installed python 3.10 but whenever I try to open a file under python it immediately closes, I'm sorry if I sound like a noob all of this is new to me and I want to get the code running! Thanks in advance and hopefully somebody can help me out!

![Python error](https://user-images.githubusercontent.com/121699507/210119655-72512abd-758f-4b88-bfb6-080b1fb8a3f9.png)

**Comment by HiddenLights** [comment]
The command is `pip install -r requirements.txt`
